docs/tutorial/statistics.md

```diff
@@ -0,0 +1,124 @@
+# Statistics
+
+Concrete analyzes all compiled circuits and calculates some statistics. These statistics can be used to find bottlenecks and compare circuits. Statistics are calculated in terms of basic operations. There are 6 basic operations in Concrete:
+
+- **clear addition:** x + y where x is encrypted and y is clear
+- **encrypted addition:** x + y where both x and y are encrypted
+- **clear multiplication:** x * y where x is encrypted and y is clear
+- **encrypted negation:** -x where x is encrypted
+- **key switch:** building block for table lookups
+- **packing key switch:** building block for table lookups
+- **programmable bootstrap:** building block for table lookups
```

Comment:
I don't know if we say "programmable bootstrap" or "programmable bootstrapping", I'll let it to @BourgerieQuentin and @aquint-zama

Reply:
Let me know before the release today!

Reply:
you can release, this one is not a big deal. @BourgerieQuentin will maybe comment, or @aquint-zama will change at doc-review time if he wants to

Reply:
seems we are using almost only "Programmable Bootstrapping" in tfhe-rs

Reply:
Updated.